### DISCUSSION TITLE
DatePicker 提问

### DISCUSSION THREAD

**Discussion by Xleihai** [element-plus/element-plus#18556] [Q&A]
https://github.com/user-attachments/assets/082e7a8a-24f7-4fc2-b023-b444a67333ab

取消掉最后一个日期会回弹到默认页 有没有办法取消这种回弹

**Comment by warmthsea** [comment]
目前看起来是不支持的，如果你想实现欢迎 PR

**Reply by Xleihai** [reply]
好的谢谢
